articles.tex

```diff
@@ -99,7 +99,9 @@
 \asubsection{Semantic Changes}
 Any semantic change to the Constitution requires the change to be proposed in writing for discussion at a House Meeting.
 Any modifications made due to the discussion are added to the written proposal and the modified proposal is posted in the House during the week.
-The final proposal is presented the following week and ballots are distributed for a ballot House vote as described in \ref{Balloted Vote}.
+The final proposal is presented the following week at the end of House Meeting.
+Attendance for the discussion on the change is not required, but only members who stay until the end of discussion will receive ballots.
```

Comment:
How does this intended to interact with quorum? 
My reading of this makes it seem like quorum is counted at the end of discussion, which would bring back a problem we had a few years ago. If there are a handful of members who object to the premise of a change, they can skip the discussion and make it impossible to hold a vote.